common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -3707,6 +3707,12 @@
     ],
     "sqlState" : "42K03"
   },
+  "PIPE_OPERATOR_SELECT_CONTAINS_AGGREGATE_FUNCTION" : {
+    "message" : [
+      "Aggregate function <expr> is not allowed when using the pipe operator |> SELECT clause; please use the pipe operator |> AGGREGATE clause instead"
```

Comment:
This is a bit odd since `|> AGGREGATE` is not added yet

Reply:
Yes, this will be tracked in a separate Jira task. The intention is that `|> SELECT` will not allow aggregate functions (ever). So this restriction on not allowing aggregate functions in `|> SELECT` will be permanent.

Until we implement `|> AGGREGATE` later, it won't be possible to use pipe operators to perform aggregation. But the feature is not complete (or enabled by default) until all the operators are implemented.